deployment/ansible/group_vars/all

```diff
@@ -30,6 +30,7 @@ postgis_package_version: "2.3.1+dfsg-1.pgdg14.04+1"
 # azavea.docker
 # 1.11.* to match AWS ECS
 docker_version: 1.11.*
+docker_options: "--storage-opt dm.basesize=20G"
```

Comment:
I see. I don't think it'll be too bad to get ECS to recognize customizations to the storage driver (unsure about Batch though).